util/genbuild_h/genbuild_h.sh

```diff
@@ -66,6 +66,10 @@ printf "#define COREBOOT_VERSION %s\n" "\"$KERNELVERSION\""
 
 #See if the build is running in a git repo and the git command is available
 printf "/* timesource: $TIMESOURCE */\n"
+
+printf "#define DASHARO_VERSION \"%s\"\n" `git describe --tags --match "asus_kgpe-d16_v[0-9]*.[0-9]*.[0-9]*" | sed 's/asus\_kgpe\-d16\_//' | sed 's/\(v[0-9]\.[0-9]\.[0-9]\+\).*/\1/'`
```

Comment:
Needs unification across other branches. OptiPlex does not use DASHARO_VERSION variables. Also these macros depend on the board name. Maybe we could get it via branch name somehow?

Reply:
This will make it more difficult to support multiple devices in a single branch which I think we want to do, eventually